src/Visualization/Python/PlotPowerMonitors.py

```diff
@@ -0,0 +1,323 @@
+                       zorder=20 + d)
+            ax.annotate(["$x$", "$y$", "$z$"][d],
+                        xy=(0, max_error[subplot_index][d]),
+                        xytext=((2 * d + 0.5) * plt.rcParams["font.size"], 0),
+                        textcoords='offset points',
+                        ha='left',
+                        va='center',
+                        bbox=dict(fc="white",
+                                  ec=props_dim[d].get("color", "black"),
+                                  pad=2.),
+                        zorder=40 + d)
+
+    # Add x-label spanning all subplots
+    ax_colspan = fig.add_subplot(111, frameon=False)
+    ax_colspan.tick_params(labelcolor='none',
+                           top=False,
+                           bottom=False,
+                           left=False,
+                           right=False)
+    ax_colspan.grid(False)
+    ax_colspan.set_xlabel("Mode number")
+
+
+def parse_step(ctx, param, value):
+    if value is None:
+        return None
+    if value.lower() == "last":
+        return -1
+    return int(value)
+
+
+@click.command()
+@click.argument("h5_files",
+                nargs=-1,
+                type=click.Path(exists=True,
+                                file_okay=True,
+                                dir_okay=False,
+                                readable=True))
+@click.option("--subfile-name",
+              "-d",
+              help=("Name of volume data subfile within the h5 files."))
+@click.option("--step",
+              callback=parse_step,
+              help=("Observation step number. Specify '-1' or 'last' "
+                    "for the last step in the file (default). "
+                    "Mutually exclusive with '--time'."))
+@click.option("--time",
+              type=float,
+              help=("Observation time. "
+                    "The observation step closest to the specified "
+                    "time is selected. "
+                    "Mutually exclusive with '--step'."))
+@click.option("--list-blocks",
+              is_flag=True,
+              help=("Print available blocks and block groups and exit."))
+@click.option("--block",
+              "-b",
+              "block_or_group_names",
+              multiple=True,
+              help=("Names of blocks or block groups to analyze. "
+                    "Can be specified multiple times."))
+@click.option("--list-vars",
+              "-l",
+              is_flag=True,
+              help="Print available variables and exit.")
+@click.option("--var",
+              "-y",
+              "vars_patterns",
+              multiple=True,
+              help=("Variables to plot. List any tensor components "
+                    "in the volume data file, such as 'Shift_x'. "
+                    "Also accepts glob patterns like 'Shift_*'."))
+@click.option("--output",
+              "-o",
+              type=click.Path(writable=True),
+              help=("Output text file"))
```

Comment:
what is saved into this text file? And where are is the image with the PM plots saved?

Reply:
The help text was just wrong, thanks for catching this as well.